packages/cli/src/commands/styles/lib/templates/css-modules/template.ts

```diff
@@ -109,7 +111,9 @@ export const typography: RenderTokenFile = (tokens, _type, { breakpoints }) => {
     `
   /* default typography tokens */
   :root {
-  ${ generateTypographyTokenBody(tokens) }
+  ${ tokens.map((token) => {
+    return `--${ token.name }: ${ token.value };`;
+  }).join('\n') }
```

Comment:
Maybe we could just use the default template (from line 19?), now that type isn't doing anything special?